aea/aea_builder.py

```diff
@@ -220,6 +220,11 @@ def pypi_dependencies(self) -> Dependencies:
             )
         return all_pypi_dependencies
 
+    def install_dependencies(self) -> None:
+        """Install extra dependencies for components."""
+        for name, d in self.pypi_dependencies.items():
+            _install_dependency(name, d)
```

Comment:
_install_dependency should be made public

Reply:
made public, moved to helper, cause aea_builder depends on cli